server/views/serviceProviderReferrals/showReferral.njk

```diff
@@ -26,6 +26,7 @@
         <p class="govuk-body">This intervention is assigned to <strong>{{ presenter.text.assignedTo }}</strong>.</p>
 
         <form method="post" action="{{ presenter.createActionPlanFormAction }}">
+          <input type="hidden" name="_csrf" value="{{csrfToken}}">
```

Comment:
What does this have to do with the rest of the PR? Can we ditch it? I think it's in George's other draft PR anyway.

Reply:
i cherry picked georges commit to fix this to make local testing palatable. does it matter which PR it's included in?

Reply:
I think it's no longer necessary now that this is handled on the intervention progress page.